doc/source/installation.rst

```diff
@@ -28,20 +28,20 @@ Find the latest tagged version of the code and check it out:
    $ git checkout <Most recent version number output by the previous command>
 
 
-Install the **bliss.core** package and its dependencies:
+Install the **ait.core** package and its dependencies:
 
 .. code-block:: bash
 
     $ pip install .
 
-From BLISS PyPi
+From AIT PyPi
 ^^^^^^^^^^^^^^^
 
-If you have access to the JPL network you can install AIT Core directly from the BLISS PyPi server.
+If you have access to the JPL network you can install AIT Core directly from the AIT PyPi server.
 
 .. code-block:: bash
 
-    $ pip install bliss-core --extra-index-url https://bliss.jpl.nasa.gov/pypi/simple/
+    $ pip install ait-core --extra-index-url https://ait.jpl.nasa.gov/pypi/simple/
```

Comment:
Should remove the reference to j.n.g. We can update the `pip install` command to install from public PyPi; we will have to update the publish the package as ait-core so that `pip install ait-core` will work. Will help #44